Runner.php

```diff
@@ -15,34 +16,28 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
-        if(isset($this->config['include_paths'])) {
-            $this->queueWithIncludePaths();
-        } else {
-            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
-        }
-
+        $this->queueWithIncludePaths();
         $this->server->process_work(false);
     }
 
     public function queueWithIncludePaths() {
         foreach ($this->config['include_paths'] as $f) {
             if ($f !== '.' and $f !== '..') {
+                if ($f == "./") {
+                    $f = "";
+                }
 
-                if (is_dir("/code$f")) {
```

Comment:
That might work.

Thoughts on using `[chdir](http://php.net/manual/en/function.chdir.php)` to set the working dir to `/code` and potentially ignore these issues?